community/contributors.md

```diff
@@ -52,6 +52,9 @@ Contributions do not constitute an official endorsement.
   * Chad Arimura - [@carimura](https://github.com/banisadr)
   * Stanley Halka - [@shalka](https://github.com/banisadr)
   * Travis Reeder - [@treeder](https://github.com/banisadr)
+* **PayPal**
+  * Vladimir Bacvanski
+  * Jem Day - [@JemDay](https://github.com/JemDay)
```

Comment:
EVERYONE - if you've submitted an issue or PR please feel free to open a PR to add your name to this doc.